better-font-awesome.php

```diff
@@ -361,11 +361,20 @@ public function create_admin_page() {
 			<form method="post" action="options.php" id="bfa-settings-form">
 			<?php
 				// This prints out all hidden setting fields.
-				settings_fields( 'option_group' );
+				settings_fields( self::SLUG );
 				do_settings_sections( self::SLUG );
 			?>
 				<p>
-					<span class="button-primary bfa-save-settings-button"><?php esc_html_e( 'Save Settings', 'better-font-awesome' ); ?></span> <img class="bfa-loading-gif" src="<?php echo esc_attr( includes_url() . 'images/spinner.gif' ); ?>" />
+					<?php
+					submit_button(
```

Comment:
Can you explain the motivation behind this change? While I imagine this might be a valuable update going forward, I'd like to minimize the amount of changes we're making and restrict them to _only_ changes that are intended to mitigate the specific security issue that's been flagged. Does this change have any impact on the security issue?

Reply:
I totally agree with you. We could move this change to another branch and limit the hotfix to fix the security issue.

Can you just merge into master the "[Implement nonce on settings form to prevent CSRF.](https://github.com/MickeyKay/better-font-awesome/pull/41/commits/7f83814254ba374f7ea698ad309cd39cfc6ea2a4)" commit and ignore the other 2? Or should I create a new pull request with that commit only?